django_object_actions/utils.py

```diff
@@ -119,8 +119,8 @@ def _get_action_urls(self):
         for action in chain(self.change_actions, self.changelist_actions):
             actions[action] = getattr(self, action)
         return [
-            # change, supports pks that are numbers or uuids
-            url(r'^(?P<pk>[0-9a-f\-]+)/actions/(?P<tool>\w+)/$',
+            # change, supports pks that are numbers, uuids and strings
+            url(r'^(?P<pk>[0-9a-zA-Z\-]+)/actions/(?P<tool>\w+)/$',
```

Comment:
thanks, I've been meaning to poke at this myself. Looking at #66 we should be looking into `.+`. I don't remember if Python's regexp's are greedy so it might have to be something like `[^/]`